### DISCUSSION TITLE
Excel error

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5581] [Help]
![image](https://user-images.githubusercontent.com/24252615/111025100-371ef780-83f3-11eb-92de-eab9c1ce75cd.png)
I'm trying to download the excel of the list, but I get an error like this, what should I do?
![image](https://user-images.githubusercontent.com/24252615/111025143-71889480-83f3-11eb-9977-32efa66d814c.png)

**Comment by edwardch** [comment]
Looks like you've updated the EPPlus package. Their new versions require licenses now.

Downgrade the package to what is in the default Serene template and it should work fine.

**Reply by havvanur160401046** [reply]
Yes it worked thank you

**Reply by erehtolleh** [reply]
@havvanur160401046 can you mark his as the answer
